solanaj/src/main/java/org/p2p/solanaj/kits/transaction/parser/SerumSwapInstructionParser.kt

```diff
@@ -0,0 +1,132 @@
+package org.p2p.solanaj.kits.transaction.parser
+
+import org.p2p.solanaj.kits.transaction.SwapDetails
+import org.p2p.solanaj.kits.transaction.network.ConfirmedTransactionRootResponse
+import org.p2p.solanaj.kits.transaction.network.meta.InstructionResponse
+import org.p2p.solanaj.programs.SerumSwapProgram
+import java.lang.Exception
+import java.math.BigInteger
+
+class SerumSwapInstructionParser {
+
+    class SerumSwapParseError(message: String) : Exception(message)
+
+    fun isTransactionContainsSerumSwap(transactionRoot: ConfirmedTransactionRootResponse): Boolean {
+        val instructions = transactionRoot.transaction?.message?.instructions.orEmpty()
+        return getSerumSwapInstructionIndex(instructions) != -1
```

Comment:
-1
local const or better global const for us

Reply:
I would rewrite it using `!= null` like its done in SwapParser